tools/ml-audio-content-moderation/terraform/main.tf

```diff
@@ -0,0 +1,253 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+provider "google" {
+  project = "${var.projectid}"
+}
+
+# Enable APIs; Must be individual resources or else it will disable all other APIs for the project.
```

Comment:
(Optional) You can consolidate these by using this module: https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services#example-usage

Reply:
For the sake of just keeping it all in one place I think we are just gonna stick with the google provider for now, but good to know that this exists!